sov-modules/sov-modules-impl/prover-incentives/src/tests.rs

```diff
@@ -0,0 +1,195 @@
+use sov_modules_api::{default_context::DefaultContext, Address, Hasher, Module, ModuleInfo, Spec};
```

Comment:
Is there specific reason to have these tests to be "unit" test and not integreation? The way it was done for bank module. It allows to test that public API's of the module is correct.

Reply:
Just for simplicity. We can move them to an integration test module, but then you have to add all of the boilerplate for call and response - and I wanted to keep this diff small

Reply:
https://github.com/Sovereign-Labs/sovereign/issues/282